lib/components/Snackbar/snackbar.stories.tsx

```diff
@@ -37,103 +59,165 @@ const meta = {
             description:
                 "Optional. Must be included if we have actionText to trigger action and close snackbar.",
         },
-        isOpen: {
-            description:
-                "Required. Controls opening and closing of snackbar. Should be set to false by default.",
-            control: "boolean",
-        },
-        onClose: {
-            description:
-                "Required. Needed to close snackbar after a certain time.",
-        },
         hasCloseButton: {
             control: "boolean",
             description: "Optional. Set to true by default.",
         },
     },
-} satisfies Meta<typeof Snackbar>;
-
-export default meta;
-
-type Story = StoryObj<typeof meta>;
-
-export const defaultSnackbar: Story = {
+    decorators: [
+        (Story) => (
+            <SnackbarProvider>
+                <SnackbarController />
+                <Story />
+            </SnackbarProvider>
+        ),
+    ],
     parameters: {
         docs: {
             story: {
-                height: "100px",
+                height: "200px",
             },
+            page: () => (
+                <>
+                    <Markdown>
+                        {`#How to use snackbar component
+First import SnackbarProvider and SnackbarController in root of your application and wrap SnackbarController inside SnackbarProvider as shown below.
+
+\`\`\`javascript
+import { SnackbarProvider } from "@providers/snackbar/snackbarProvider";
+import { SnackbarController } from "@components/Snackbar";
+
+ReactDOM.createRoot(document.getElementById("root")!).render(
+    <React.StrictMode>
+        <ThemeProvider>
+            <div className="quill__background--primary__type--base">
+                <SnackbarProvider>
+                    <SnackbarController />
+                    .....Other Components
+                </SnackbarProvider>
+            </div>
+        </ThemeProvider>
+    </React.StrictMode>,
+);
+\`\`\`
+
+To add snackbar anywhere in your application, import addSnackbar function from useSnackbar hook and customize your snackbar by providing desired props.
+
+\`\`\`javascript
+import { useSnackbar } from "@hooks/useSnackbar";
+
+<Button 
+    size="lg" 
+    label={"Add Snackbar"}
+    onClick={() => {
+        addSnackbar({
```

Comment:
```suggestion
        useSnackbar.addSnackbar({
```

Reply:
ah yeah i forgot to put this in documentation

const { addSnackbar } = useSnackbar();